.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+- package-ecosystem: pip
+  directory: "/"
+  schedule:
+    interval: weekly
+  open-pull-requests-limit: 10
+  commit-message:
+    prefix: "[noissue]"
```

Comment:
Ugh this is what will activate Dependabot on all forks.

Any new reason to add this file?

Reply:
yes? who will manage the automatic updates of the released deps?https://github.com/pulp/pulpcore/blob/main/docs/plugins/plugin-writer/concepts/index.rst#declaring-dependencies

I don;t think it will activate on forks, can you point me to the config you then expect to see?

Reply:
This is rather a _big_ annoyance. Not a reason to stop doing this.

Reply:
BTW: https://github.com/dependabot/dependabot-core/issues/2804